kvmagent/kvmagent/test/shareblock_testsuite/test_sharedblock_ha.py

```
@@ -0,0 +1,153 @@
+        self.condition_dict = {1: "success", 0: "fail", -1:"no_way"}
+        self.fencer_result_dict = {1: "trigger", 0:"no_trigger"}
+        self.sanlock_io_timeout = 5
+        self.management_network_ok = True
+        self.fencer_triggered = 0
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    def mock_zsblk_agent(self):
+        def mock_func(uri, body=None, headers={}, fail_soon=False, print_curl=False):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)